sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -0,0 +1,43 @@
+import os
+import sbol3
+sbol3.set_namespace("https://testing.sbol3.genbank/")
```

Comment:
This does not belong in the converter. A user should set this to the namespace they want. The converter should honor what the user has chosen.

Reply:
Alright, I have provided namespace as a parameter to the function with a default.